docs/reference/troubleshooting/data/restore-from-snapshot.asciidoc

```diff
@@ -0,0 +1,13 @@
+[[restore-from-snapshot]]
+== Restore from snapshot
+
+Elasticsearch is using snapshots to store a copy of your data outside a cluster. You can restore a snapshot to recover
+indices and data streams for which there are no copies of the shards in the cluster. This can happen after a deletion or
+hardware failure.
+
+IMPORTANT: Restoring the missing data requires you to have a usable backup of the affected indices and data streams. Please do not
```

Comment:
```suggestion
IMPORTANT: Restoring the missing data requires you to have a backup of the affected indices and data streams. Please do not
```

Reply:
Replaced the word `usable` in a way that is consistent with the rest of the document.